aten/src/ATen/CMakeLists.txt

```diff
@@ -264,11 +247,12 @@ IF(USE_CUDA AND NOT USE_ROCM)
 ENDIF()
 
 IF(USE_ROCM)
- ### Link in the ROCm libraries BLAS / RNG.
+ ### Link in the ROCm libraries BLAS / RNG / FFT.
  FIND_LIBRARY(ROCBLAS_LIBRARY rocblas HINTS ${ROCBLAS_PATH}/lib)
  FIND_LIBRARY(HIPRAND_LIBRARY hiprand HINTS ${HIPRAND_PATH}/lib)
+ FIND_LIBRARY(ROCFFT_LIBRARY rocfft HINTS ${ROCFFT_PATH}/lib)
 
- list(APPEND ATen_CUDA_DEPENDENCY_LIBS ${ROCBLAS_LIBRARY} ${HIPRAND_LIBRARY})
+ list(APPEND ATen_CUDA_DEPENDENCY_LIBS ${ROCBLAS_LIBRARY} ${HIPRAND_LIBRARY} ${ROCFFT_LIBRARY})
```

Comment:
We don't need this here. In LoadHIP.CMake, the find_package(rocfft) will automatically find the libraries for us.

Though, make sure to add the var ${rocfft_INCLUDE_DIRS}  to Caffe2_HIP_INCLUDES inside cmake/Dependencies.cmake.